gobblin-runtime/src/main/java/gobblin/runtime/JobContext.java

```diff
@@ -96,6 +99,13 @@ public JobContext(Properties jobProps, Logger logger) throws Exception {
         previousJobState.getPropAsInt(ConfigurationKeys.JOB_FAILURES_KEY, 0));
 
     if (GobblinMetrics.isEnabled(jobProps)) {
+      Configuration configuration = new Configuration();
+      String flowId = configuration.get("azkaban.flow.flowid");
+
+      if (StringUtils.isNotBlank(flowId)) {
```

Comment:
Strings.isNullOrEmpty does not check white spaces.
it returns false for " ", I want it to be true.